.circleci/config.yml

```diff
@@ -1,4 +1,4 @@
-# config file version: 1.0.3
+# config file version: 1.1.0
```

Comment:
Why did this jump a minor version?

Reply:
Because we added steps to build all targets